README.rst

```diff
@@ -273,6 +273,7 @@ build metadata is equality.
     >>> Version('1.0.0+build1') in SimpleSpec('==1.0.0+build2')  # Include build in checks
     False
 
+When using '+' in the spec text only operators '==' and '!=' are supported whilst any variant of lower/greater is not.
```

Comment:
@AlexanderStohr I felt that this topic was already addressed in the previous sentence:

> Build metadata has no ordering; thus, the only meaningful comparison including build metadata is equality.

Maybe I should add a few examples of comparing versions higher in the README? Or some more details in this example block?

Reply:
I felt it was not obvious but only implicit by leaving out such examples.

Reply:
comparing metadata can be done in principle by using a textual (ascii) or a numeric comparision. Right now its just up to the one that wants to do it - to do it on his own.